src/components/subtitlesettings/subtitlesettings.js

```diff
@@ -111,6 +114,19 @@ function onSubtitleModeChange(e) {
     view.querySelector('.subtitles' + this.value + 'Help').classList.remove('hide');
 }
 
+function onSubtitleStyleChange(e) {
+    const view = dom.parentWithClass(e.target, 'subtitlesettings');
+
+    const subtitleStylingHelp = view.querySelectorAll('.subtitleStylingHelp');
+    for (let i = 0, length = subtitleStylingHelp.length; i < length; i++) {
+        subtitleStylingHelp[i].classList.add('hide');
```

Comment:
```suggestion
    for (const elem of view.querySelectorAll('.subtitleStylingHelp')) {
        elem.classList.add('hide');
```
or you can also use `forEach` - it won't give any extra code after transpilation, iirc.